hyperstart.go

```diff
@@ -204,7 +204,7 @@ func (h *hyper) linkPauseBinary(podID string) error {
 
 	pausePath := filepath.Join(pauseDir, pauseBinName)
 
-	return os.Symlink(h.config.PauseBinPath, pausePath)
+	return fileCopy(h.config.PauseBinPath, pausePath)
```

Comment:
Also, particularly since this function has been problematic in the past, how about introducing a `TestCopyPauseBinary()` function? We could even check copying to a new fs with a tmp mount and also ensure the `src` and `dest` have the expected different inode numbers.